dataflow/README.md

```diff
@@ -33,53 +36,20 @@ The following instructions help you prepare your Google Cloud project.
 
 1. [Enable billing](https://cloud.google.com/billing/docs/how-to/modify-project).
 
-1. [Enable the Dataflow API](https://console.cloud.google.com/flows/enableapi?apiid=dataflow).
-
-1. Create a service account JSON key via the
-   [*Create service account key* page](https://console.cloud.google.com/apis/credentials/serviceaccountkey),
-   or via the `gcloud` command line tool.
-   Here is how to do it through the *Create service account key* page.
+1. Enable the Dataflow API.
 
-   * From the **Service account** list, select **New service account**.
-   * In the **Service account name** field, enter a name.
-   * From the **Role** list, select **Project > Owner** **(*)**.
-   * Click **Create**. A JSON file that contains your key downloads to your computer.
+   <button><a href="https://console.cloud.google.com/flows/enableapi?apiid=dataflow">
+      Click here to enable the API
+   </a></button>
 
-   Alternatively, you can use `gcloud` through the command line.
+1. Authenticate to your Google Cloud project.
```

Comment:
Can we update with permissions the user needs to be able to run these samples?

Reply:
For most cases, the default application credentials should work without any problems. Dataflow permissions are a little complicated since they're different for launching jobs and for the workers. I linked to https://cloud.google.com/dataflow/docs/concepts/security-and-permissions where there is more information about the permissions and roles needed.